code/datums/json_savefile.dm

```diff
@@ -0,0 +1,145 @@
+#define JSON_SAVEFILE_PARSE_CUTOFF_FLAG "||__||__||"
+#define JSON_SAVEFILE_PARSE_REPLACE_COMMA "|||_|||"
```

Comment:
also wait what is this actually for

Reply:
these seem to be sentry values they are using in the export_text parser to mark the end of sections, cutoff_flag seems to be an EOF of sorts

i'm pretty sure there is a way to `for` thru a savefile so these should get deleted once that gets made

Reply:
FOUND IT:

```dm
for (var/key in savefile.dir)
	savefile[key] >> value
	data[key] = value
```

Reply:
yeaaaah do this instead especially to kill the ExportText stuff